### PR TITLE
Add scroiset SSH key

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: classes/system/openssh/server/team/stacklight.yml [hunk 1]
```diff
@@ -9,6 +9,13 @@ parameters:
           full_name: Ales Komarek
           home: /home/newt
           email: mail@newt.cz
+        scroiset:
+          enabled: true
+          name: newt
```
Comment (elemoine) [inline classes/system/openssh/server/team/stacklight.yml]: newt?

File: classes/system/openssh/server/team/stacklight.yml [hunk 2]
```diff
@@ -20,6 +27,13 @@ parameters:
...
+        scroiset:
+          enabled: true
+          public_keys:
+          - ${public_keys:scroiset}
+          user: ${linux:system:user:scroiset}
   public_keys:
     newt:
       key: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC3odU+3V2uDA2ptAFL9hrJRPNEEdAyztWOZFQ5Oyd9oerTGOU3p4xmrgWWjfKFKbYGhiiIUcYAol5PkTfKukGEkkjCHYA1t023soCaaAj85wCZCnw2zQNAziwxTYmAzTqgxiSvtZNMMrtJvFHRIRDzJ3M1lV0prWNWkMM1/3FAd4W49y6VT3fkMCo8uqG7CfGdgR2DgBCxf9KaNPfW5eDEPOgmE5lK8tVSEI6T+Cg7hbcTf4lFYnlFBnlQgp/0JstsM4Vbwb4B34LOpOsf2S8rrWk2xQMjwaMHXkc2s/E8iW3F5nVFuyEXYISFQIiAHw8dzC6CHgLcyHUVWwznKawZ newt@newt-dev1
+    scroiset:
+        key: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDUfVxx6qi4fay4znw8M6yLJJcRr3pdvPPihTAszioFJP9V/GBbqfkofTfeKdpdvJ4j25p40boiDt43Ek7LfcRmKMP9+2SEfk9W7ec/umM8Mer+h4ocnShVQm69weELVUfr9q4G+qWf14ANc9D097bclqQ6FP/cjy8HodVPgQ+i1lpMjwP6xvAAERJJH353lCFsxkh2N8aOi9YcP9M2lQeKWM+eYFsdcmTFAPHbgPq0K4ma6/YXw5UibEBClYu1u4OJTFZSI3z8kERb2cU0aFGYAduiynDMBKM7y7YAoksgBOVprq0huEMFUqJ3vsrZbPn55GIpzmBga+EGnNbSCadt swann@scroiset
```
Comment (elemoine) [inline classes/system/openssh/server/team/stacklight.yml]: indent?

**Comment by elemoine** [comment]
LGTM.

**Comment by pupapaik** [comment]
can you please do rebase?

**Pull request closed without merging**